src/System.Net.WebSockets.Client/tests/CloseTest.cs

```diff
@@ -257,6 +256,7 @@ public class CloseTest : ClientWebSocketTestBase
             }
         }
 
+        [ActiveIssue(20362, TargetFrameworkMonikers.Uap)]
```

Comment:
I've updated #20362 explaining that the UWP ClientWebSocket implementation mirrors the .NET Framework behavior, leading to this test failing. Isn't the .NET Framework behavior the "gold standard", though?

I'm wondering whether this test method should be updated to pin the two types of behaviors instead of having an active issue. It seems to me like .NET Framework's behavior isn't buggy; it's just different from non-UWP .NET Core. Thoughts?